### PR TITLE
Add R113 blog post

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+<h3 id="metrics">1.1 Prometheus metrics support</h3>
+
+Thanks to [LiveIntent][liveintent], the Scala Stream Collector now publishes
+[Prometheus][prometheus] metrics to the `/metrics` endpoint. You'll find the following metrics
+published at this endpoint:
+
+- `http_requests_total`: the total count of requests
+- `http_request_duration_seconds`: the time spent handling requests
+
+Which you'll be able to slice and dice by endpoint, method and/or response code.
```
Comment (goodits) [inline _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md]:
```suggestion
You will be able to slice and dice the metrics by endpoint, method and/or response code.
```

File: _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+- Deal deletion
+- Contact creation
+- Contact change
+- Contact deletion
+- Company creation
+- Company change
+- Company deletion
+
+Improvements have also been made to the [Marketo](https://www.marketo.com/) and
+[CallRail](https://www.callrail.com/) integrations.
```
Comment (goodits) [inline _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md]: By Peter?

File: _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+<h3 id="post">2.2 POST support in the API request enrichment</h3>
+
+It is now possible to use `POST` requests to interact with the API leveraged in the API request
```
Comment (goodits) [inline _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md]: Can you add what this is used for, Ben?

File: _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+Note, that those changes are also true for Stream Enrich for Kafka through the
+`enrich.streams.sourceSink.{producerConf, consumerConf}` configurations.
+
+Thanks a lot to [Sven Pfenning][sven] and [Mirko Prescha][mirko] for those two awesome features!
+
+<h3 id="ssc-other">1.3 Other improvements</h3>
+
+For people using [the do not track cookie feature](https://snowplowanalytics.com/blog/2018/08/21/snowplow-r109-lambaesis-real-time-pipeline-upgrade/#dnt)
+of the Scala Stream Collector, [LiveIntent][liveintent] has improved the feature by letting you
+specify a regex for the cookie value.
```
Comment (goodits) [inline _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md]: Can you think of a use case to add here?

File: _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+[Mike][miike] from [Snowflake Analytics][sa] has introduced a configurable `Access-Control-Max-Age`
+header which lets clients cache the results of `OPTIONS` request, resulting in fewer requests and
+faster `POST` requests: no need to make a preflight request if the result is already cached.
+
+<h2 id="sce">2. Scala Common Enrich improvements</h2>
+
+<h3 id="hubspot">2.1 HubSpot webhook integration</h3>
+
+[Peter Zhu][misterpig] from [Snowflake Analytics][sa] built the [HubSpot][hubspot] webhook
+integration from scratch for this release, huge props to him!
```
Comment (goodits) [inline _posts/2019-02-18-snowplow-r113-filitosa-real-time-pipeline-improvements/index.md]:
```suggestion
integration from scratch for this release. Huge props to Peter!
```

**Comment by BenFradet** [comment]
Thanks, fixed :+1: 

@Mandelliant could we add this one to next Wednesday's list? Thanks!

**Comment by Mandelliant** [comment]
You got it, @BenFradet!

**Comment by BenFradet** [comment]
https://maven.twttr.com/ is down so we can't publish the version of spark enrich for now. This means we'll delay the blog post till next week.

**Pull request merged**